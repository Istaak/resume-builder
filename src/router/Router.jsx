// Third-party dependencies
import React from 'react';
import { BrowserRouter, Switch, Route } from 'react-router-dom';

// Own constants/actions/components
import Home from 'containers/Home/Home';
import NotFound from 'containers/NotFound/NotFound';

const Router = () => (
  <BrowserRouter>
    <Switch>
      <Route exact path="/">
        <Home />
      </Route>

      <Route component={NotFound} />
    </Switch>
  </BrowserRouter>
);

export default Router;
