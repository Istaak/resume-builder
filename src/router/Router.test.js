// Third-party dependencies
import React from 'react';
import ReactTestRender from 'react-test-renderer';

// Own constants/actions/components
import Router from './Router';

describe('Router: ', () => {
  test('should render correctly', () => {
    const component = ReactTestRender.create(<Router />);
    const render = component.toJSON();
    expect(render).toMatchSnapshot();
  });
});
