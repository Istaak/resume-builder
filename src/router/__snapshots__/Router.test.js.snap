// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Router:  should render correctly 1`] = `
<h1>
  Resume Builder
</h1>
`;
